src/patternfly/sass-utilities/mixins.scss

```diff
@@ -201,3 +201,18 @@
     #{$value}: #{$prop};
   }
 }
+
+// Animate tab focus removal
+@mixin pf-animate-remove-tab-focus($element, $delay: $pf-global--TransitionDuration) {
+  @keyframes visibility-hidden {
```

Comment:
I missed this before. Can we rename this to something specific with "pf" so it doesn't conflict with something a product might add? Any concern with having it match the mixin name? Also what do you think about a more generic name for this? Maybe "pf-animate-visibility-hidden"?